runtimes/standalone/src/lib.rs

```diff
@@ -436,6 +436,32 @@ impl did::Config for Runtime {
 	type WeightInfo = ();
 }
 
+parameter_types! {
+	pub const DidLookupDeposit: Balance = KILT;
+}
+
+impl pallet_did_lookup::Config for Runtime {
+	type Event = Event;
+	type Signature = Signature;
+	type Signer = AccountPublic;
+	type DidIdentifier = DidIdentifier;
+
+	type Currency = Balances;
+	type Deposit = DidLookupDeposit;
+
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type EnsureOrigin = did::EnsureDidOrigin<DidIdentifier, AccountId>;
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type OriginSuccess = did::DidRawOrigin<AccountId, DidIdentifier>;
```

Comment:
I'm not a fan of this. 🤔  It's longer and i don't see the benefit. Also makes it a bit more obscure since you don't know which type is used there immediately.

Reply:
Sure, this is also ok.